src/config.h

```diff
@@ -28,13 +28,3 @@
 		#define BUILD_TYPE "Debug"
 	#endif
 #endif
-
-#define BUILD_INFO \
```

Comment:
50b2185ced9dc64e054c
#6461

Reply:
As far as I've understood, this only affects putting a build date there.

If the version changes, the `cmake_config_githash.h` is updated and then `version.cpp` will also be recompiled. Am I missing something?

Reply:
Makes sense. We can investigate this if #defines other than the build date are affected.